code/src/containers/dashboard-tabs/components/filter-record-accordion/FilterRecordAccordion.scss

```diff
@@ -0,0 +1,49 @@
+@import "@design-system";
+
+.filter-record-accordion {
+  border: 0;
+  border-radius: 0 !important; // to override mui styles
```

Comment:
remove the comment. Hope everyone know about `!important`

Reply:
I won't remove it. We should take into account that our project can be transferred to another team and they will not understand that it means

Reply:
> I won't remove it. We should take into account that our project can be transferred to another team and they will not understand that it means

If someone don't know what is the `!important` he can google it. Don't litter the code. We don't have any information about the project further

Reply:
We already have places where this approach is used, so i will not do it. It is the best way to use comments in our code so we should take advantage of them.

![image](https://github.com/user-attachments/assets/7eb2afb4-1dc7-4f0a-a72a-ac98cbfbc265)

Take a look, it is actively used by our teammates

Reply:
> We already have places where this approach is used, so i will not do it. It is the best way to use comments in our code so we should take advantage of them.
> 
> ![image](https://private-user-images.githubusercontent.com/109922101/348966165-7eb2afb4-1dc7-4f0a-a72a-ac98cbfbc265.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjExMTE4MjcsIm5iZiI6MTcyMTExMTUyNywicGF0aCI6Ii8xMDk5MjIxMDEvMzQ4OTY2MTY1LTdlYjJhZmI0LTFkYzctNGYwYS1hNzJhLWFjOThjYmZiYzI2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcxNlQwNjMyMDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZjBjZDg4MGQzYjIwNmZmZWJiMTlmNzVjYmUyYTM2NDU4ZmU2NWQ3ZDlmNDUyNjZjMWVjYmQwOGMwYzMzZDI4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.3m9_RidreppXmx5z9yEuf4OD8KKFi0X0sDlAfP1wO68)
> 
> Take a look, it is actively used by our teammates

I wrote to remove the comment. thats it